CRITICAL - 2020-08-18 06:07:51 --> Class 'App\Controllers\M_Usuario' not found
#0 C:\xampp\htdocs\arthur\system\CodeIgniter.php(914): App\Controllers\Login->Logar()
#1 C:\xampp\htdocs\arthur\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#2 C:\xampp\htdocs\arthur\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\arthur\index.php(45): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2020-08-18 06:08:07 --> You must set the database table to be used with your query.
#0 C:\xampp\htdocs\arthur\system\Model.php(1206): CodeIgniter\Database\BaseConnection->table(NULL)
#1 C:\xampp\htdocs\arthur\system\Model.php(1714): CodeIgniter\Model->builder()
#2 C:\xampp\htdocs\arthur\app\Controllers\Login.php(27): CodeIgniter\Model->__call('AcessaSistema', Array)
#3 C:\xampp\htdocs\arthur\system\CodeIgniter.php(914): App\Controllers\Login->Logar()
#4 C:\xampp\htdocs\arthur\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\arthur\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\arthur\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2020-08-18 06:09:46 --> Undefined variable: _SESSION
#0 C:\xampp\htdocs\arthur\app\Controllers\Login.php(35): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 35, Array)
#1 C:\xampp\htdocs\arthur\system\CodeIgniter.php(914): App\Controllers\Login->Logar()
#2 C:\xampp\htdocs\arthur\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\arthur\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\arthur\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-08-18 06:10:05 --> Undefined variable: _SESSION
#0 C:\xampp\htdocs\arthur\app\Controllers\Login.php(35): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 35, Array)
#1 C:\xampp\htdocs\arthur\system\CodeIgniter.php(914): App\Controllers\Login->Logar()
#2 C:\xampp\htdocs\arthur\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\arthur\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\arthur\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-08-18 06:48:48 --> Undefined variable: _SESSION
#0 C:\xampp\htdocs\arthur\app\Controllers\Login.php(35): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 35, Array)
#1 C:\xampp\htdocs\arthur\system\CodeIgniter.php(914): App\Controllers\Login->Logar()
#2 C:\xampp\htdocs\arthur\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\arthur\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\arthur\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2020-08-18 06:50:36 --> Undefined index: ID_SETOR
#0 C:\xampp\htdocs\arthur\app\Models\M_Usuario.php(184): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 184, Array)
#1 C:\xampp\htdocs\arthur\app\Controllers\Login.php(27): App\Models\M_Usuario->AcessarSistema('1314', '25d55ad283aa400...')
#2 C:\xampp\htdocs\arthur\system\CodeIgniter.php(914): App\Controllers\Login->Logar()
#3 C:\xampp\htdocs\arthur\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#4 C:\xampp\htdocs\arthur\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\arthur\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-08-18 06:50:52 --> Cannot use a scalar value as an array
#0 C:\xampp\htdocs\arthur\app\Models\M_Usuario.php(174): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Cannot use a sc...', 'C:\\xampp\\htdocs...', 174, Array)
#1 C:\xampp\htdocs\arthur\app\Controllers\Login.php(27): App\Models\M_Usuario->AcessarSistema('1314', '25d55ad283aa400...')
#2 C:\xampp\htdocs\arthur\system\CodeIgniter.php(914): App\Controllers\Login->Logar()
#3 C:\xampp\htdocs\arthur\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#4 C:\xampp\htdocs\arthur\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\arthur\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2020-08-18 06:51:11 --> Undefined offset: 0
#0 C:\xampp\htdocs\arthur\app\Controllers\Login.php(29): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'C:\\xampp\\htdocs...', 29, Array)
#1 C:\xampp\htdocs\arthur\system\CodeIgniter.php(914): App\Controllers\Login->Logar()
#2 C:\xampp\htdocs\arthur\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\arthur\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\arthur\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
